// Copyright 2011, Tencent Inc.
// Author: Hangjun Ye <hansye@tencent.com>
// Xiaokang Liu <hsiaokangliu@tencent.com>
// Eric Liu <ericliu@tencent.com>

import "poppy/rpc_option.proto";

package rpc_examples;

message EchoRequest {
    required string user = 1;
    required string message = 2;
}

message EchoResponse {
    required string user = 1;
    required string message = 2;
}

// The server side serivce.
service EchoServer {
    rpc Echo(EchoRequest) returns(EchoResponse);
}
